### DISCUSSION TITLE
Inject value in `miniRuntime`

### DISCUSSION THREAD

**Discussion by 321paranoiawhy** [jpudysz/react-native-unistyles#280] [Feature Requests]
From the doc, we can get some info from `miniRuntime`, but how can we add value in it so the value can be shared accross the app, I mean register the value in `miniRuntime` as soon as possible

**Comment by 321paranoiawhy** [comment]
since already have `orientation` key in `miniRuntime `, what if I want to add key `isLandscape` in `miniRuntime`? so that we don't have code like this in `createStyleSheet`:

```ts
const stylesheet = createStyleSheet((theme, miniRuntime) => {
  const isLandscape = miniRuntime.orientation === 'landscape';

  return {
    text: {
      color: isLandscape ? 'red : 'blue'
    }
  };
});
```

**Comment by jpudysz** [comment]
Interesting idea! 
Immediately I thought about themes, but themes are evaluated only once, so they wont change the value. I will think about it 👨‍💻

**Reply by 321paranoiawhy** [reply]
> Interesting idea! Immediately I thought about themes, but themes are evaluated only once, so they wont change the value. I will think about it 👨‍💻

Hope you can find a way to achieve that, thanks. Or, how can we do styles according the orientation? Is there a best practice?

**Reply by jpudysz** [reply]
Actually there is.. maybe that's the solution. If you don't register own breakpoints, Unistyles will auto inject two of them. They're called: "portrait" and "landscape".

**Reply by ghost** [reply]
I would also love to to use it like the breakpoints! Just: `backgroundColor: {dark: "red", light: "blue", myTheme: "green"}`. Also a Nesting would be helpful:  `backgroundColor: {dark: {landscape: "red", portrait: "orange"}, light: "blue", myTheme: "green"}`
